### PR TITLE
GL-1950. Expose read length parameter in wgs metrics

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

File: pipelines/broad/dna_seq/germline/single_sample/wgs/test_inputs/Plumbing/G96830.NA12878.json [hunk 1]
```diff
@@ -56,5 +56,7 @@
     "agg_preemptible_tries": 3
   },
 
-  "WholeGenomeGermlineSingleSample.UnmappedBamToAlignedBam.CheckContamination.disable_sanity_check": true
+  "WholeGenomeGermlineSingleSample.UnmappedBamToAlignedBam.CheckContamination.disable_sanity_check": true,
+  "WholeGenomeGermlineSingleSample.CollectWgsMetrics.read_length": 300,
```
Comment (gbggrant) [inline pipelines/broad/dna_seq/germline/single_sample/wgs/test_inputs/Plumbing/G96830.NA12878.json:60]: TODO - set me back.

**Review by ldgauthier — APPROVED** [review]
To the best of my knowledge, this looks reasonable (and I think we've discussed it before?) but Chris K. gets the last word.

**Review by wjdingman1 — APPROVED** [review]

**Review by kachulis — APPROVED** [review]
Looks good!

**Comment by gbggrant** [comment]
add to whitelist

**Comment by gbggrant** [comment]
retest this please

**Review by nikellepetrillo — APPROVED** [review]

**Pull request merged**